editors/sc-ide/core/sc_server.cpp

```diff
@@ -392,9 +411,29 @@ void ScServer::onScLangReponse( const QString & selector, const QString & data )
 	else if (selector == startDumpOSCSelector) {
         mActions[DumpOSC]->setChecked(true);
     }
-	else if (selector == stopDumpOSCSelector) {
+    else if (selector == stopDumpOSCSelector) {
         mActions[DumpOSC]->setChecked(false);
     }
+	else if (selector == recordingDurationSelector) {
+        bool ok;
+        float duration = data.mid(1, data.size() - 2).toFloat(&ok);
+        if (ok) {
+            mRecordTime = (int) duration;
+            updateRecordingAction();
+        }
+    }
+    else if (selector == startRecordingSelector) {
+        setRecording(true);
+    }
+    else if (selector == startRecordingSelector) {
+        setRecording(true);
+    }
+    else if (selector == pauseRecordingSelector) {
+        pauseRecording(true);
+    }
+    else if (selector == stopRecordingSelector) {
+        setRecording(false);
+    }
```

Comment:
Yes, indeed I currently have the problem that for some reason, that code breaks. I'm a bit at loss why this is, 
````
s.boot;

s.mute; // see the red “m”
s.unmute;

s.record; // this sends “recordingStarted” to the IDE
s.stopRecording; // this sends “recordingStopped”

s.mute; // now this doesn’t display any more
s.unmute;
````

This even happens when I comment out the above code. I checked carefully and it is nothing in sclang.